src/spriteset_map.cpp

```diff
@@ -40,6 +41,13 @@ Spriteset_Map::Spriteset_Map() {
 		character_sprites.push_back(EASYRPG_MAKE_SHARED<Sprite_Character>(i->second.get()));
 	}
 
+	Game_Vehicle* vehicle;
+	for (int i = 1; i <= 3; ++i) {
+		vehicle = Game_Map::GetVehicle((Game_Vehicle::Type) i);
+		if (vehicle->IsInCurrentMap())
```

Comment:
The vehicles will only be visible if they're on the map when the spriteset is created then. Since `Game_Vehicle::GetVisible` checks if they're in the current map, how about just removing this check and pushing them on unconditionally?